web/src/engine/websites/templates/ZeistManga.ts

```diff
@@ -0,0 +1,43 @@
+import { Chapter, DecoratableMangaScraper, Manga, type MangaPlugin } from '../../providers/MangaPlugin';
+import * as Common from '../decorators/Common';
+import { FetchJSON, FetchWindowScript } from '../../platform/FetchProvider';
+
+type FeedResults = {
+    feed: {
+        entry: {
+            link: {
+                rel: string,
+                href: string,
+                title: string
+            }[],
+        }[]
+    }
+}
+
+function PageLinkExtractor(image: HTMLImageElement): string {
+    return image.src.replace(/(\/s(\d+[^/]*)(\/[^/]+$))/, '/s0$3');
```

Comment:
Two out of the Three capture groups in the regular expression seems to be unused and should be removed